RELEASE-NOTES.txt

```diff
@@ -1,8 +1,13 @@
 Unreleased
 ---
+
+
+1.96.0
+---
 * [*] Add disabled style to `Cell` component [https://github.com/WordPress/gutenberg/pull/50665]
+* [**] Fix undo/redo history when inserting a link configured to open in a new tab [https://github.com/WordPress/gutenberg/pull/50460]
 * [**] Disable details settings for not belonged VideoPress videos [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5782]
-* [***] Enable VideoPress block on Android (only available in Simple WPCOM sites) [#5780]
+* [***] Enable VideoPress block on Android (only available in Simple WPCOM sites) [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780]
 * [**] Tapping any type of nested block moves focus to the nested block directly, rather than requiring multiple taps to navigate down each nesting levels. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5781]
```

Comment:
Hey there 👋 I just noticed this change is not being included in the main apps release notes, is there a reason for that? It's not a major change but thought I'd ask.

Reply:
Ah, good catch. I overlooked this entry when reviewing the PRs, sorry for that. That said, I think we should incorporate it, maybe it's not too late to incorporate it into the release notes 🤞 .

Reply:
I've created a couple of PRs to update the release notes into version `22.5`:
- [WP-Android](https://github.com/wordpress-mobile/WordPress-Android/pull/18551)
- [WP-iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/20767)

Reply:
Approved! Thank you for adding it and wrangling the PRs! 🙇